cmake/helpers.cmake

```diff
@@ -0,0 +1,39 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: MIT
+
+#
+# helpers.cmake -- helper functions for top-level CMakeLists.txt
+#
+
+# Sets ${ret} to version of program specified by ${name} in major.minor format
+function(get_program_version_major_minor name ret)
+	execute_process(COMMAND ${name} --version
```

Comment:
This file is also using tabs instead of 4 spaces making it inconsistent with the rest of the project.

Reply:
Done.